ucfopen/Materia-Widget-Dev-Kit
# dev/3.0.3

MWDK 3.0.3 dev branch. Includes required updates to support the new `0.4.0` version of `materia-widget-dependencies`.

### How to test:

1. Check out MWDK and switch to this branch locally
2. Run `yarn install` followed by `yarn build`
3. Run `yarn link`
4. Navigate to a local widget repository and make sure you've installed node dependencies already via `yarn install`
5. Run `yarn link "materia-widget-development-kit"` to replace the local MWDK node module with a symlink to your local copy
6. Run `yarn start` to init the MWDK for the widget as usual
7. Test the widget locally at `localhost:8118`
